interactive-calculator/src/index.js

```diff
@@ -0,0 +1,17 @@
+import React from 'react';
+import ReactDOM from 'react-dom';
+import './index.css';
+import App from './App';
+import * as serviceWorker from './serviceWorker';
+
+ReactDOM.render(
+  <React.StrictMode>
+    <App />
+  </React.StrictMode>,
+  document.getElementById('root')
+);
+
+// If you want your app to work offline and load faster, you can change
+// unregister() to register() below. Note this comes with some pitfalls.
+// Learn more about service workers: https://bit.ly/CRA-PWA
+serviceWorker.unregister();
```

Comment:
Did you actually make use of the serviceWorker? I honestly can't tell if this is just boilerplate in there or not, but if you didn't use it, I think it's best to delete it. 

(this is literally the only thing I can think of to say other than WOW so great job!!)

Reply:
Thats the create-react-app boilerplate. I should have deleted that.

I've used it in gatsby before, but not with cra. Not sure if its much different